addons/mail/static/src/components/messaging_menu/messaging_menu.js

```diff
@@ -149,6 +141,10 @@ class MessagingMenu extends Component {
         }
     }
 
+    _onClickDesktopTabButton(ev) {
+        this.messagingMenu.onClickDesktopTabButton(ev.currentTarget.dataset.tabId);
```

Comment:
forward whole ev to model and let model handle the logic (commit: Move onClickDesktopTabButton)